src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -2,39 +2,84 @@
 
 public class IntLinkedList {
 
-    Node last;
-    int i;
-
-    public void push(int i) {
-        if (last == null)
-            last = new Node(i);
-        else {
-            last.next = new Node(i);
-            last.next.prev = last;
-            last = last.next;
-        }
-    }
-
-    public boolean isEmpty() {
-        return last == null;
-    }
-
-    public boolean isFull() {
-        return false;
-    }
-
-    public int top() {
-        if (isEmpty())
-            return -1;
-        return last.value;
-    }
-
-    public int pop() {
-        if (isEmpty())
-            return -1;
-        int ret = last.value;
-        last = last.prev;
-        return ret;
-    }
+	private static final int ERROR = -1;
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.